pkg/tests/test_rpm.spec

```diff
@@ -18,6 +18,7 @@ This is a package description.
 %build
 
 %install
+cp -r ./tests %{buildroot}/
```

Comment:
This worries me. I can't evaluate the impact for other .spec files.
Do you have working .spec files that do not do something like this.?

Reply:
Since the ./tests folder exists before running rpmbuild, and is not an output of the `%build` stage, I copy the files into the buildroot for subsequent use in the `%files` stage. This seems to be inline with RPM conventions.